Dockerfile.py3

```diff
@@ -133,4 +135,5 @@ USER ${USER}
 # install python-for-android from current branch
 RUN virtualenv --python=python3 venv \
     && . venv/bin/activate \
+    && pip3 install -U Cython \
```

Comment:
Isn't the one installed system wide enough?
https://github.com/kivy/python-for-android/blob/v2019.07.08/Dockerfile.py3#L126

Reply:
Hm true I may have interpreted that one error situation wrong, let me try it might work without that :+1:

Reply:
Yup system-wide is enough, apparently :joy: well-spotted!

Reply:
I understand, thanks for investigating!
Then maybe we should ditch the system one?

Reply:
Ouch, I just saw the build is failing for Python2.
https://travis-ci.org/kivy/python-for-android/jobs/567481453
```
2943  RAN: /usr/bin/python2 -m Cython.Build.Cythonize ./jnius/jnius.pyx
2944
2945  STDOUT:
2946/usr/bin/python2: No module named Cython.Build
```
So that's properly weird and annoying. Basically it means if we want to support the "Python2 basic" apk build in the CI, we do need the system cython on python2. While only having the venv cython3. That would look kinda inconsistent in the Dockerfile, but with some comments that should make it.
So basically we want to have:
```
RUN pip2 install --upgrade Cython==0.28.6
```
And also in the venv:
```
&& pip3 install --upgrade Cython==0.28.6
```
By the way, let's take the opportunity to make them consistent.

- uppercase `Cython` because that's it's naming on pypi
- long argument `--upgrade` because we love explicit
- explicit "officially p4a tested" version `0.28.6` in all places (eventually via variable)